src/coreclr/jit/emit.cpp

```diff
@@ -3894,15 +3894,24 @@ void emitter::emitDispJumpList()
     unsigned int jmpCount = 0;
     for (instrDescJmp* jmp = emitJumpList; jmp != nullptr; jmp = jmp->idjNext)
     {
+#if defined(TARGET_ARM64)
+        if ((jmp->idInsFmt() == IF_LARGEADR) || (jmp->idInsFmt() == IF_LARGELDC))
+        {
+            printf("IG%02u IN%04x %3s[%u] -> %s\n", jmp->idjIG->igNum, jmp->idDebugOnlyInfo()->idNum,
+                   codeGen->genInsDisplayName(jmp), jmp->idCodeSize(), getRegName(jmp->idReg1()));
+        }
+        else
+        {
+            printf("IG%02u IN%04x %3s[%u] -> IG%02u\n", jmp->idjIG->igNum, jmp->idDebugOnlyInfo()->idNum,
+                   codeGen->genInsDisplayName(jmp), jmp->idCodeSize(),
+                   ((insGroup*)emitCodeGetCookie(jmp->idAddr()->iiaBBlabel))->igNum);
+        }
+#else
         printf("IG%02u IN%04x %3s[%u] -> IG%02u %s\n", jmp->idjIG->igNum, jmp->idDebugOnlyInfo()->idNum,
                codeGen->genInsDisplayName(jmp), jmp->idCodeSize(),
                ((insGroup*)emitCodeGetCookie(jmp->idAddr()->iiaBBlabel))->igNum,
-#if defined(TARGET_XARCH)
-               jmp->idjIsRemovableJmpCandidate ? " ; removal candidate" : ""
-#else
-               ""
+               jmp->idjIsRemovableJmpCandidate ? " ; removal candidate" : "");
 #endif
-               );
```

Comment:
This xarch check is needed until your PR adding this for arm is finished idjIsRemovableJmpCandicate is xarch only.

`/__w/1/s/src/coreclr/jit/emit.cpp:3913:21: error: no member named 'idjIsRemovableJmpCandidate' in 'emitter::instrDescJmp'
               jmp->idjIsRemovableJmpCandidate ? " ; removal candidate" : "");`

Reply:
Ah, right. Forgot about that. Thanks!